db/migrate/20160719204409_add_smoker_to_profile.rb

```diff
@@ -0,0 +1,5 @@
+class AddSmokerToProfile < ActiveRecord::Migration
+  def change
+    add_column :profiles, :smoker, :text
```

Comment:
How can we match on someone being a smoker if this is a text field? I suggest having a boolean for smoker / non-smoker.